packages/cyberstorm/src/components/componentStyles/PackageFlag.module.css

```diff
@@ -0,0 +1,29 @@
+.root {
+  display: flex;
+  flex-direction: row;
+  gap: var(--gap-3xs);
+  align-items: center;
+  justify-content: center;
+
+  padding: var(--padding-4xs) var(--padding-3xs);
+  border: var(--size-6xs) solid var(--border-color);
```

Comment:
I don't think a border is needed for this component, is there a reason for it? To be clear `border-radius` and `border` are unrelated.

Reply:
Not actually sure why and  where I do get these numbers. Could argue that I saw em on Figma on first rotation, but still might be just my mistake. Will fix this 👌

Reply:
Yup, that actually fixed it looking bit weird! Nice catch cap 👊